openmls/src/group/core_group/kat_welcome.rs

```diff
@@ -0,0 +1,259 @@
+use openmls_rust_crypto::OpenMlsRustCrypto;
```

Comment:
Can you add a markdown header like in the other test vectors?

(from here: https://github.com/mlswg/mls-implementations/blob/main/test-vectors.md#welcome)